Tasks/CopyFilesV2/task.json

```diff
@@ -79,6 +79,15 @@
             "required": false,
             "helpMarkDown": "Flatten the folder structure and copy all files into the specified target folder.",
             "groupName": "advanced"
+        },
+        {
+            "name": "keepTimestamp",
+            "type": "boolean",
+            "label": "Update Target Timestamp",
```

Comment:
Both the label and helpMarkDown fields are kinda misleading. Really what this is doing is retaining the original file's timestamp, and **not** update it. We should probably update these descriptions **or** keep the descriptions, set the defaultValue to true, and reverse the logic in copyfiles.ts to comply with the naming. I'd lean towards the first option though.

Reply:
I've changed this to preserve from the other conversation, that seemed like more clear language than keep or update.